avaframe/ana3AIMEC/ana3AIMEC.py

```diff
@@ -360,7 +444,8 @@ def postProcessAIMEC(rasterTransfo, pLim, newRasters, cfgPath, cfgFlags):
 
     return resAnalysis
 
-def postProcessAIMECReport(rasterTransfo, pLim, newRasters, cfgPath, cfgFlags):
+
+def postProcessAIMECReport(rasterTransfo, newRasters, cfgSetup, cfgPath, cfgFlags):
```

Comment:
What ist the difference of this function to `postProcessAIMEC` ? There seems to be a lot of code overlap.